### DISCUSSION TITLE
Script Console Menu Button Crash [Stellarium 0.21.3 - Linux] (repeatable)

### DISCUSSION THREAD

**Discussion by ModMerc** [Stellarium/stellarium#2277] [General]
Clicking on the "Open", "Save", "Clear" menu bar buttons, and the button located next to the "Include dir" text box causes Stellarium to freeze for 8 seconds, then close.

The "Preprocess Scripts", "Run" buttons, and the "Execute" drop down menu can be clicked without crashing the application.

I'm using Linux Mint 20.3, Cinnamon Desktop with
kernel version:  5.4.0-962201191901-generic

**Comment by gzotti** [comment]
Further discussed in Issue #2278.
